pom.xml

```diff
@@ -155,6 +155,7 @@
     <dependency>
       <groupId>org.apache.commons</groupId>
       <artifactId>commons-lang3</artifactId>
+      <version>3.18.0</version>
```

Comment:
Needed with Quarkus up to 3.24.x, but 3.25.0 would raise this dep to 3.18.0 without override.